binary-codec/types/amount.go

```diff
@@ -231,7 +231,7 @@ func SerializeIssuedCurrencyValue(value string) ([]byte, error) {
 }
 
 // Serializes an issued currency code to its bytes representation. The currency code can be 3 allowed string characters, or 20 bytes of hex
-func serializeIssuedCurrencyCode(currency string) ([]byte, error) {
+func SerializeIssuedCurrencyCode(currency string) ([]byte, error) {
```

Comment:
Is there a reason this is now being exported?

Reply:
not anymore! changed